lib/auth-handler.js

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
I believe that using kebab-case for filenames is not very common for javascript projects. why not use camelCase instead?